collision_clearance: 0.08

collision_links:
  r_gripper_l_finger_link:
    link_radius: 0.03
    link_clearance: 0.13
  r_gripper_r_finger_link:
    link_radius: 0.03
    link_clearance: 0.13
  r_gripper_palm_link:
    link_radius: 0.05
    link_clearance: 0.10
  r_forearm_link:
    link_radius: 0.065
    link_clearance: 0.08
  r_upper_arm_link:
    link_radius: 0.07
    link_clearance: 0.08

reference_frame: base_link

collision_space:
  size_x: 1.2
  size_y: 2.0
  size_z: 2.2
  origin_x: -0.1
  origin_y: -1.5
  origin_z: -0.3
  resolution: 0.01
